src/Components/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+interface ButtonsProps {
```

Comment:
be consistent
State in App, Props in ListOfGoods, but ButtonsProps props here, name it just Props too